pkg/api/openapi/minder/v1/minder.swagger.json

```diff
@@ -3351,6 +3413,27 @@
       },
       "description": "GitType defines the git data ingester."
     },
+    "v1Invitation": {
+      "type": "object",
+      "properties": {
+        "role": {
+          "type": "string",
+          "description": "role is the role that would be assigned if the user\naccepts the invitation."
+        },
+        "email": {
+          "type": "string",
+          "description": "email is the email address of the invited user.  This is\npresented as a convenience for display purposes, and does\nnot affect who can accept the invitation using the nonce."
+        },
+        "project": {
+          "type": "string",
+          "description": "project is the project to which the user is invited."
+        },
+        "nonce": {
```

Comment:
does this mean that we are returning the nonce/invitation code, so basically the someone with admin permissions is able to accept someone else's invitation if they request the code through the API? :thinking:

Reply:
Yes, this does mean that a user with the ability to grant access to a project can create an invitation and immediately use the code to accept that invitation and grant themselves that permission.  They could also just grant themselves that permission without an invitation.

This is to support cases like "share the invitation via slack" or for self-hosted Minder environments that haven't set up email sending but want to invite another user to a project.